components/feature/downloads/src/test/java/mozilla/components/feature/downloads/AbstractFetchDownloadServiceTest.kt

```diff
@@ -778,7 +782,75 @@ class AbstractFetchDownloadServiceTest {
 
         service.setDownloadJobStatus(downloadJobState, COMPLETED)
         assertEquals(COMPLETED, service.getDownloadJobStatus(downloadJobState))
-        testDispatcher.advanceTimeBy(750)
+        testDispatcher.advanceTimeBy(PROGRESS_UPDATE_INTERVAL)
         assertEquals(1, shadowNotificationService.size())
     }
+
+    @Test
+    fun `DownloadJobState-getSecondsSinceTheLastNotificationUpdate`() = runBlocking {
```

Comment:
nit: Can we follow the existing test name pattern here? This `abc-def` looks a bit weird to me :).